includes/widgets/text-editor.php

```diff
@@ -126,17 +126,25 @@ protected function register_controls() {
 			]
 		);
 
-		$text_columns = range( 1, 10 );
-		$text_columns = array_combine( $text_columns, $text_columns );
-		$text_columns[''] = esc_html__( 'Default', 'elementor' );
-
 		$this->add_responsive_control(
 			'text_columns',
 			[
 				'label' => esc_html__( 'Columns', 'elementor' ),
 				'type' => Controls_Manager::SELECT,
 				'separator' => 'before',
-				'options' => $text_columns,
+				'options' => [
+					'' => esc_html__( 'Default', 'elementor' ),
+					'1' => esc_html__( '1', 'elementor' ),
```

Comment:
Why we need to translate numbers?

Reply:
@arielk see my comment above.

Languages like Arabic and Persian that have their own numbers.

Reply:
@rami-elementor Does WordPress, WooCommerce or Yoast use this method as well?